src/bundle/Resources/views/profile_image/initials.svg.twig

```diff
@@ -1,4 +1,6 @@
 <symbol id="profile_image" viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg">
-    <ellipse ry="15" rx="15" cy="16" cx="16" fill="{{ background }}" stroke="#000000" stroke-width="3" />
-    <text x="5" y="22" font-size="16px" fill="{{ text }}">{{ initials }}</text>
+    <ellipse ry="15" rx="15" cy="16" cx="16" fill="{{ background }}" stroke="{{ text_color }}" stroke-width="1" />
```

Comment:
Maybe we should not make that same mistake twice and use different variable name for stroke color as it is not text related, right?